test/language/expressions/class/private-method-brand-check-multiple-evaluations-of-class-realm-function-ctor.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2019 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Every new evaluation of a class creates a different brand (private method)
+esid: sec-privatefieldget
+info: |
+  ClassTail : ClassHeritage { ClassBody }
+    ...
+    11. Let proto be ObjectCreate(protoParent).
+    ...
+    31. If PrivateBoundIdentifiers of ClassBody contains a Private Name P such that the P's [[Kind]] field is either "method" or "accessor",
+      a. Set F.[[PrivateBrand]] to proto.
+    ...
+
+  PrivateBrandCheck(O, P)
+    1. If O.[[PrivateBrands]] does not contain an entry e such that SameValue(e, P.[[Brand]]) is true,
+      a. Throw a TypeError exception.
+features: [class, class-methods-private]
+---*/
+
+let classStringExpression = `
+return class C {
+  #m() { return 'test262'; }
+
+  access(o) {
+    return o.#m();
+  }
+}
+`;
+
+let createAndInstantiateClass = function () {
+  return new (new ($262.createRealm().global.Function)(classStringExpression));
```

Comment:
Ditto for this returning a class expression instead of an instance of the evaluated class.